Client/linux/LinuxBluetoothConnector.cpp

```diff
@@ -100,9 +102,14 @@ std::vector<BluetoothDevice> LinuxBluetoothConnector::getConnectedDevices()
   for (auto &adapter : adapter_paths)
   {
     printf("%s\n", adapter.c_str());
-    std::string name = dbus_get_property(connection, adapter.c_str(), "Name");
-    std::string address = dbus_get_property(connection, adapter.c_str(), "Address");
-    res.push_back({.name = name, .mac = address});
+	  // Filter out if the device does not have the expected UUID
```

Comment:
The code here filters out devices that don't have the service UUID. This will remove any issue with connecting to an incompatible device.